libs/features/personalization/personalization.js

```diff
@@ -51,6 +51,10 @@ const DATA_TYPE = {
   TEXT: 'text',
 };
 
+export const MERCH_CARD_COLLECTION_SELECTOR = 'in-card-collection';
```

Comment:
I think this constant should not be here, personalization.js should not know about blocks needing delayed MEP based personalization.